blueman/main/SpeedCalc.py

```diff
@@ -3,13 +3,13 @@
 
 
 class SpeedCalc:
-    def __init__(self, moving_avg=3):
+    def __init__(self, moving_avg: float = 3) -> None:
```

Comment:
Shouldn't this cause an error, 3 being an int while the annotation says float? If this is indeed a float let's make it clear with the default of 3.0.

Reply:
mypy is fine with using `int` values where `float` is expected. I can change the default to `3.0`.

Reply:
I think this would make more sense and consistant. Even if python and mypy are fine with it.